caffe2/mobile/contrib/arm-compute/core/rewrite_net.cc

```diff
@@ -62,7 +62,7 @@ static void insertCopyFromGLOp(NetDef& predictNet, const std::string& cpu_blob)
   auto* op = predictNet.add_op();
   op->set_name("CopyFromGL");
   op->set_type("CopyFromGL");
-  op->add_input(cpu_blob);
+  op->add_input(cpu_blob + "_M");
```

Comment:
Why?

Reply:
We can use an temporary variable to first copy out the gpu memory, and then reuse the blob to store the cpu result, but I feel that might be less efficient comparing to copy from the gpu memory directly to cpu memory(which is contained in another blob)?